website/blog/2023-10-25-mongodb-sorting-scalar.md

```diff
@@ -0,0 +1,168 @@
+---
+slug: mongodb-sorting-scalar
+title: How MongoDB Sorting Works for Scalar Values
+authors: [chi]
+description: >
+  In this blog post, we explore how MongoDB sorting works for scalar values.
+keywords:
+  - mongodb sorting
+  - bson comparison order
+image: /img/blog/post-cover-image.jpg
+tags: [community, product, tutorial]
```

Comment:
@Fashander are these tags sensible? I kept keywords let me know if you think they are note necessary.

Reply:
Removed keywords.